src/annotator/highlighter.js

```diff
@@ -260,6 +261,10 @@ export function highlightRange(range, cssClass = 'hypothesis-highlight') {
     /** @type {HighlightElement} */
     const highlightEl = document.createElement('hypothesis-highlight');
     highlightEl.className = cssClass;
+    //here is where we can call another function to get the color for the user
+    //color-->index number in the 'user-color' class
+    highlightEl.classList.toggle('user-color'+Math.round(colorNum), true);
```

Comment:
And to change the color without classes, it's something like `highlightEl.style.backgroundColor = color`